neqo-http3/src/connection.rs

```diff
@@ -1962,4 +1962,115 @@ mod tests {
         assert_eq!(hconn.state(), Http3State::GoingAway);
         hconn.close(now(), 0, String::from(""));
     }
+
+    #[test]
+    fn test_stream_fin_wo_data() {
+        let (mut hconn, mut neqo_trans_conn, _) = connect_and_receive_control_stream(true);
+        let request_stream_id = hconn
+            .fetch(
+                &"GET".to_string(),
```

Comment:
The ergonomics of this are terrible.  Can we change the API to make this easier to consume?

Reply:
open an issue?